### DISCUSSION TITLE
Chlorine is not in database

### DISCUSSION THREAD

**Discussion by defencedog** [equinor/neqsim#867] [Ideas]
I am a chloralkali engineer & has found Neqsim very beneficial. Can you please add chlorine in database?

**Comment by EvenSol** [comment]
Thanks for the comment. I have made a pull request for adding the chlorine parameters. If you have more pure component properties we should add to the comp table, please inform about that. See:

https://github.com/equinor/neqsim/pull/871

**Comment by EvenSol** [comment]
Chlorine is now available as a component in the most recent  release of NeqSim.
